app/presenters/flood_risk_engine/enrollment_presenter.rb

```diff
@@ -38,6 +38,11 @@ def correspondence_contact_name
       title
     end
 
+    def organisation_registration_number
+      return unless enrollment.organisation && enrollment.organisation.registration_number.present?
+      enrollment.organisation.registration_number
+    end
+
     private
```

Comment:
I do like delegation in places but the above feels much harder to read and understand exactly what is going on than just a plain method